src/imio/esign/browser/static/esign.css

```
@@ -1,3 +1,28 @@
+/* Add-to-custom-esign-session overlay table */
+#add-to-custom-esign-session-form .sessions-table {
+    table-layout: fixed;
+    width: 100%;
+    word-wrap: break-word;
```

Stylelint (17.11.1)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)